db.go

```diff
@@ -49,7 +50,14 @@ type DBResponse struct {
 }
 
 func newDBRunner(name, dsn string) (*dbRunner, error) {
-	db, err := dburl.Open(normalizeDSN(dsn))
+	var db *sql.DB
+	var err error
+	if strings.HasPrefix(dsn, "sp://") {
```

Comment:
Can you support `spanner://` as well as https://github.com/xo/dburl?

Reply:
@k1LoW 
I researched and found that `spanner://` is also supported. (`sp://` was the alias that had been set)
So I fixed like this. https://github.com/k1LoW/runn/pull/482/commits/c690fad5e24856df4430b708ab46984ba7c144f2
What do you think?

- [dburl/scheme.go at b1661a2a1fc75d11485c718e112f09ed4778010f · xo/dburl](https://github.com/xo/dburl/blob/b1661a2a1fc75d11485c718e112f09ed4778010f/scheme.go#L95)
- [Go Playground - The Go Programming Language](https://go.dev/play/p/DRI8beVGrWP)